src/api-reference/fi-service/financial-integration-service.markdown

```diff
@@ -0,0 +1,1124 @@
+
+The Financial Integration API works only with the new [Authentication API](https://developer.concur.com/api-reference/authentication/apidoc.html).
+
+```
+HEADER:   "Authorization: Bearer {YOUR ACCESS TOKEN}"
+```
+
+## GET a list of Financial Transactions based on a search criteria
+
+Making a GET request to the financial transactions API will return a list of available financial documents, ready to be processed.
+
+```
+GET   /financialintegration/fi/v1/companies/{companyId}/transactiontypes/{docType}/transactions
+```
+
+### Parameters
+
+| Name | Type | Format | Description |
+| --- | --- | --- | --- |
+| `companyId` | `string` | `path` | **Required** Company Outtask Id in Concur |
+| `docType` | `string` | `path` | **Required** The financial document type to return. VALUES: expense; invoice; cashadvance. Only one type of transactions can be retrieved at a time |
+| `OuttaskCompanyId` | `string` | `header` | **Required** Outtask Company Id in Concur |
+| `entityid` | `string` | `header` | Concur unique code for the entity |
+| `offset` | `string` | `query` | Starting page offset |
+| `limit` | `int32` | `query` | Number of records to return per page (default 100) |
+| `docId` | `string` | `query` | The transaction unique identifier, it could be expense report ID, payment request ID or Cash Advance ID. If specified, a single document which matches docId is returned. |
+| `ignoreDocumentStatus` | `string` | `query` | Ignores the financial documents status. FORMAT: yes or no. if yes, a document is returned regardless of status. If no, only documents that have not been acknowledged/confirmed are returned |
+| `systemId` | `string` | `query` | The external system ID that processed the document |
+
+### Response schema
+
+#### Financial Documents
+
+| Name | Type | Format | Description |
+| --- | --- | --- | --- |
+| `fIDocuments` | `Array` | [Financial document](#fidocument) | The result collection |
+| `nextPage` | `string` | - | The URI of the next page of results |
+| `numberReturned` | `int32` | - | Number of documents returned in current page |
+| `offset` | `string` | - | Next page offset |
+
+#### Financial Document
+
+| Name | Type | Format | Description |
+| --- | --- | --- | --- |
+| `id` | `string` | `-` | The unique identifier for the document |
+| `docType` | `string` | - | Transaction type. Expense, invoice or cashadvance |
+| `companyId` | `string` | - | Unique identifier for the company in Concur |
+| `entityId` | `string` | - | Unique Identifier for the entity in Concur |
+| `companyUuid` | `string` | `UUID` | Unique UUID for the company in Concur |
+| `erpSystemId` | `string` | - | The external System ID that processed the document |
+| `document` | `Array` | - | The JSON financial document. Review the [FI sample documents](#examples) below |
+| `docStatus` | `string` | - | The financial document status. VALUES: READY, ACKNOWLEDGED, POSTING_CONFIRMED_SUCCESS, POSTING_CONFIRMED_FAILURE. |
+
+_Example Request:_
+
+cURL:
```

Comment:
No references to specific tooling allowed. Keep it super simple.

Reply:
Take a look at the revised version Imene submitted today addressing your input (last commit 991304a). For the entityID and CompanyOutask ID, Teresa’s team are working on a v2 of this API to hide those details to the customers.